openff/bespokefit/schema/targets.py

```diff
@@ -20,6 +23,140 @@
 from openff.bespokefit.utilities.pydantic import SchemaBase
 
 
+def _check_connectivity(
+    qcschema: QCEMolecule,
+    name: str,
+    fragment: Molecule,
+    expected_connectivity: Optional[Set[Tuple[int, int]]] = None,
```

Comment:
Can we not use the connectivity in the qcschema molecule as our `expected_connectivity`? As its made from the input openff molecule [here](https://github.com/openforcefield/openff-toolkit/blob/817563a06d8c583a66bea74df278b1eddbaa3124/openff/toolkit/topology/molecule.py#L4316) it should always be correct and the same and it removes the need for the `_off_mol_to_connectivity` function.